specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/2016-01-01/DownloadedProduct.json

```diff
@@ -0,0 +1,238 @@
+                ],
+                "operationId": "DownloadedProducts_Get",
+                "parameters": [
+                    {
+                        "$ref": "AzureBridge.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ActivationNameParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ProductNameParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/DownloadedProductResource"
+                        }
+                    },
+                    "404": {
+                        "description": "Not Found"
+                    }                    
+                },
+                "produces": [
+                    "application/json"
+                ],
+                "consumes": [
+                    "application/json"
+                ]
+            },
+            "delete": {
+                "x-ms-examples": {
+                    "Deletes the specified downloaded product.": {
+                        "$ref": "./examples/DownloadedProduct/Delete.json"
+                    }
+                },
+                "description": "Delete a downloaded product.",
+                "tags": [
+                    "DownloadedProducts"
+                ],
+                "operationId": "DownloadedProducts_Delete",
+                "parameters": [
+                    {
+                        "$ref": "AzureBridge.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ActivationNameParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ProductNameParameter"
+                    },
+                    {
+                        "$ref": "AzureBridge.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "202": {
+                        "description": "Accepted",
+                        "schema": {
+                            "$ref": "#/definitions/DownloadedProductResource"
+                        }
+                    },
+                    "404": {
+                        "description": "Not Found"
+                    }                    
```

Comment:
long running operation should also have status code 200 or 204.

Reply:
Done.